test/github-issues/9173/issue-9173.ts

```diff
@@ -0,0 +1,57 @@
+import "reflect-metadata"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { DataSource, Table } from "../../../src"
+import { View } from "../../../src/schema-builder/view/View"
+
+describe("github issues > #9173 missing typeorm_metadata", () => {
```

Comment:
you can look at [this test](https://github.com/typeorm/typeorm/tree/master/test/github-issues/2693) to see how to test migrations